app/javascript/src/app/controllers/search_autocomplete_controller.js

```diff
@@ -37,6 +39,19 @@ export default class extends Controller {
     this.resultsTarget.removeEventListener('click', this.onResultsClick);
   }
 
+  initCommandK() {
+    hotkeys('command+k', function(event, handler) {
+      // Prevent the default refresh event under WINDOWS system
+      event.preventDefault();
+      jQuery('#btn-search-toggle')
+        .data('HSUnfold')
+        .show();
+      setTimeout(function() {
+        jQuery('#search-input').focus();
```

Comment:
Probably better to use explicit Stimulus targets rather than `id`s to access elements.

If the desired elements aren't part of the Stimulus controller, I generally wrap them in getter methods that match the Stimulus targets interface.

```js

get someElementTarget() {
  return document.getElementById('some-element');
}
```

Reply:
See the `useCapture` argument here: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener